src/main/resources/assets/minebound/models/item/magic/test_spell.json

```diff
@@ -1,6 +1,6 @@
 {
     "parent": "item/generated",
     "textures": {
-      "layer0": "minebound:item/magic/test_spell"
+      "layer0": "minebound:item/__"
     }
   }
```

Comment:
You can probably just remove this file, no? Or is it being used to generate model files?

Reply:
I used it for generating the others; it's not 100% necessary, but while there's still more of the models to generate we can probably leave it